tests/integration/test_causal_regression_modeling_cfg.yaml

```diff
@@ -12,7 +12,9 @@ dataset:
     add_eos_token_to_answer: false
     add_eos_token_to_prompt: false
     add_eos_token_to_system: false
-    answer_column: regression_label
+    answer_column: 
+    - regression_label
+    - regression_label
```

Comment:
This is something not possible in the UI, I'd suggest to update the dataframe to have multiple regression label columns. Adding one column twice should probably even result in an error.